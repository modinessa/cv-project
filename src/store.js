import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
