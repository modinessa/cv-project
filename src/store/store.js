import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});