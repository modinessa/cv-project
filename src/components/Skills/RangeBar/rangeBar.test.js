import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import { store } from '../../../store/store';
import { RangeBar } from './index';

const TEST_NAME = 'skill';
const TEST_RANGE = 50;

describe('RangeBar', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<RangeBar
				name={TEST_NAME}
				range={TEST_RANGE}
				/>
			</Provider>
		)).toMatchSnapshot();
	})
})