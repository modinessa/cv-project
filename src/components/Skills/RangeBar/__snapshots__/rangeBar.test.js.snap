// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RangeBar renders component 1`] = `
<div
  className="range-bar"
  style={
    Object {
      "width": "calc(50% - 30px)",
    }
  }
>
  skill
</div>
`;
